docker/nginx-custom.conf

```diff
@@ -0,0 +1,38 @@
+server {
+    listen       8080;
+    # same config for localhost, dev and prod environments
+    server_name  localhost 
+                swedgene.scilifelab-2-dev.sys.kth.se
+                genomes.scilifelab.se;
+    absolute_redirect off;
+
```

Comment:
Since the root path is the same across multiple locations, you can move the root directive to the server context for better maintainability:
root /usr/share/nginx/html;
This way, all location blocks inherit the root unless overridden.

Reply:
Thanks, I didn't know that could be done. I have done that now as part of commit 7500e00953d8cfd91204e9ff65c5230533ecb59f